examples/Examples/PIsData.hs

```diff
@@ -93,3 +98,45 @@ pInteger =
     , testCase "PlutusTx.fromData (pdata 100) ≡ Just 100" $
         testPFromDataCompat @PInteger 100
     ]
+
+pUnit :: HasTester => TestTree
+pUnit =
+  testGroup
+    "Builtin PIsData instances: PUnit"
+    [ testCase "pfromData (pdata ()) ≡ ()" $
+        testPToFrom @PUnit ()
+    , testCase "pfromData (PlutusTx.toData ()) ≡ ()" $
+        testPFromDataCompat @PUnit ()
+    , testCase "PlutusTx.fromData (pdata ()) ≡ Just ()" $
+        testPDataCompat @PUnit ()
+    ]
+
+pPair :: HasTester => TestTree
+pPair =
```

Comment:
Naming

Reply:
Chosen to be consistent with rest of this module. Perhaps they should all be renamed to something like `ppairTests` and similar?

Reply:
I do think so. This was a mistake that we didn't notice.

Reply:
Updated.